metaphor/looker/config.py

```diff
@@ -53,6 +53,9 @@ class LookerRunConfig(BaseConfig):
     # Alternative base url to build the entity source URL
     alternative_base_url: Optional[str] = None
 
+    # LookML explores & views folder name
+    explore_view_folder_name: str = "LookML models"
```

Comment:
```suggestion
    explore_view_folder_name: str = "LookML Models"
```

For consistency with "LookML Dashboards"